<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ReflEventWidget</class>
  <widget class="QWidget" name="ReflEventWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>959</width>
        <height>346</height>
      </rect>
    </property>
    <property name="font">
      <font>
        <pointsize>8</pointsize>
        <bold>false</bold>
      </font>
    </property>
    <property name="windowTitle">
      <string>Event Widget</string>
    </property>
    <layout class="QVBoxLayout" name="eventModeMainLayout">
      <property name="margin">
        <number>5</number>
      </property>
      <item>
        <widget class="QGroupBox" name="uniformGroup">
          <property name="title">
            <string>Uniform slicing</string>
          </property>
          <layout class="QVBoxLayout" name="uniformLayoutContainer">
            <item>
              <layout class="QGridLayout" name="uniformLayout0">
                <item row="0" column="0">
                  <widget class="QRadioButton" name="uniformEvenButton">
                    <property name="checked">
                      <bool>true</bool>
                    </property>
                    <attribute name="buttonGroup">
                      <string notr="true">slicingOptionsButtonGroup</string>
                    </attribute>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="QLineEdit" name="uniformEvenEdit">
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>10</width>
                        <height>10</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="0" column="2">
                  <widget class="QLabel" name="uniformEvenLabel">
                    <property name="text">
                      <string>even time slices</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="3">
                  <spacer name="uniformEvenSpacer">
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                  </spacer>
                </item>
                <item row="1" column="0">
                  <widget class="QRadioButton" name="uniformButton">
                    <attribute name="buttonGroup">
                      <string notr="true">slicingOptionsButtonGroup</string>
                    </attribute>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QLineEdit" name="uniformEdit">
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>10</width>
                        <height>10</height>
                      </size>
                    </property>
                  </widget>
                </item>
                <item row="1" column="2">
                  <widget class="QLabel" name="uniformLabel">
                    <property name="text">
                      <string>(sec) slices</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="3">
                  <spacer name="uniformSpacer">
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="customGroup">
          <property name="title">
            <string>Custom slicing</string>
          </property>
          <layout class="QHBoxLayout" name="customLayout">
            <item>
              <widget class="QRadioButton" name="customButton">
                <attribute name="buttonGroup">
                  <string notr="true">slicingOptionsButtonGroup</string>
                </attribute>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="customLabel">
                <property name="text">
                  <string> Python list (sec)</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="customEdit">
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>10</width>
                    <height>10</height>
                  </size>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="customSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>50</height>
            </size>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <tabstops/>
  <customwidgets/>
  <resources/>
  <connections/>
  <buttongroups>
    <buttongroup name="slicingOptionsButtonGroup"/>
  </buttongroups>
</ui>